3p/ampcontext.js

```diff
@@ -312,8 +313,21 @@ export class AbstractAmpContext {
       this.setupMetadata_(this.win_.name);
     }
   }
-}
 
+  /**
+   * Send 3p error to parent iframe
+   */
+  errorReport() {
+    this.win_.addEventListener('error', event => {
+      if (!!event.error) {
+        this.client_.sendMessage(MessageType.USER_ERROR, dict({
+          'error': event.error,
+          'message': event.error.message,
```

Comment:
is `event` the `ErrorEvent`? if so should it be `event.message` here?

Reply:
I was thinking event.error returns error so I can get message from that error

Reply:
I honestly don't know. Can you make sure?